snapcraft/tests/plugins/test_catkin.py

```diff
@@ -249,6 +260,55 @@ def test_schema(self):
         self.assertThat(underlay_run_path, Contains('type'))
         self.assertThat(underlay_run_path['type'], Equals('string'))
 
+        # Check rosinstall-files property
+        self.assertTrue('rosinstall-files' in properties,
+                        'Expected "rosinstall-files" to be included in '
+                        'properties')
+
+        rosinstall_files = properties['rosinstall-files']
+        self.assertTrue('type' in rosinstall_files,
+                        'Expected "type" to be included in "rosinstall-files"')
+        self.assertTrue('default' in rosinstall_files,
+                        'Expected "default" to be included in '
+                        '"rosinstall-files"')
+        self.assertTrue('minitems' in rosinstall_files,
+                        'Expected "minitems" to be included in '
+                        '"rosinstall-files"')
+        self.assertTrue('uniqueItems' in rosinstall_files,
+                        'Expected "uniqueItems" to be included in '
+                        '"rosinstall-files"')
+        self.assertTrue('items' in rosinstall_files,
+                        'Expected "items" to be included in '
+                        '"rosinstall-files"')
```

Comment:
I would drop all those assertion messages, and just use self.assertIn instead.
Not a big deal, or -1, just my preference.